spec/schema.json

```diff
@@ -118,12 +240,89 @@
     }
   },
   "definitions": {
+    "Data": {
+      "type": "object",
+      "properties": {
+        "partnership_name": {
+          "type": "string"
+        },
+        "name_ending": {
+          "type": "string",
+          "enum": [
+            "Limited Partnership",
+            "LP",
+            "L.P.",
+            "Partneriaeth Cyfyngedig",
+            "PC",
+            "P.C."
+          ]
+        },
+        "partnership_type": {
+          "type": "string",
+          "enum": [
+            "LP",
+            "PFLP",
+            "SLP",
+            "SPFLP"
+          ]
+        }
+      }
+    },
+    "LimitedPartnershipIncorporationGetResponse": {
+      "type": "object",
+      "required": [
+        "etag",
+        "kind"
+      ],
+      "properties": {
+        "etag": {
+          "type": "string"
+        },
+        "kind": {
+          "type": "string"
+        },
+        "sub_resources": {
+          "type": "object",
+          "properties": {
+            "general_partners": {
+              "type": "array",
+              "items": {
+                "properties": {
+                  "id": {
```

Comment:
Again, I don't think ids are returned for the partners. Maybe we can just use a comment here or reference what the general partner GET returns?

Reply:
referenced the returned object type

Reply:
changed to return the existing partner submission